modules/rosa-sharing-vpc-dns-and-roles.adoc

```diff
@@ -0,0 +1,72 @@
+// Module included in the following assemblies:
+//
+// * networking/rosa-shared-vpc-config.adoc
+:_content-type: PROCEDURE
+[id="rosa-sharing-vpc-dns-and-roles_{context}"]
+= Creating your DNS and cluster creation roles
+
+After the *VPC-owning AWS account* user creates a virtual private cloud, subnets, and an IAM role for sharing the VPC resources, the *cluster-creating AWS account* user must reserve an _openshiftapps.com_ DNS domain and create Operator roles to communicate back to the *VPC-owning AWS account*.
```

Comment:
Same comment here about highlighting for user role names. In the OCP docs, role names are typically in monospace font. I'm not sure if the conventions in the ROSA docs are different. Also, consider using monospace instead of italics for "openshiftapps.com" unless that is a user-replaced value.